javascript-arrays/main.js

```diff
@@ -0,0 +1,30 @@
+var colors = ['red', 'white', 'blue'];
+
+console.log('value of colors[0]: ', colors[0]);
+console.log('value of colors[1]: ', colors[1]);
+console.log('value of colors[2]: ', colors[2]);
+
+var USFlag = 'America is';
```

Comment:
make sure these variables start with lowercase letters. Uppercase is only for Classes